media-libs/libmpdclient/libmpdclient-2.14.recipe

```diff
@@ -3,26 +3,29 @@ DESCRIPTION="Music Player Daemon (MPD) is a flexible, powerful, server-side appl
 for playing music. Through plugins and libraries it can play a variety \
 of sound files while being controlled by its network protocol."
 HOMEPAGE="http://www.musicpd.org/"
-COPYRIGHT="2003-2017 The Music Player Daemon Project"
+COPYRIGHT="2003-2018 The Music Player Daemon Project"
 LICENSE="BSD (3-clause)"
 REVISION="1"
 SOURCE_URI="https://github.com/MusicPlayerDaemon/libmpdclient/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="e1583fbcae89ad543a6cc44caeaf16b31e709831f5b7eac4413f78c54a21db95"
+CHECKSUM_SHA256="262f1ff13e6ee11cca2ef93f2eda10ec6953a7b8e628572645e707fe530fbbb6"
 
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="?x86"
 
+libVersion="$portVersion"
```

Comment:
Are you sure upstream bumped the soname from libmpdclient.so.**2.0.13** to libmpdclient.so.**2.14** (i.e. without a zero) ?
If the bump is from `libmpdclient.so.2.0.13` to `libmpdclient.so.2.0.14` then the correct libVersion would be:
```diff
-libVersion="$portVersion"
+libVersion="2.0.14"
```

Reply:
There is no zero in the soname.

Reply:
OK, thanks.